pennylane/qinfo/classical_fisher.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Differentiable classical fisher information"""
+
+import pennylane as qml
+import pennylane.numpy as pnp
+
+import jax
+import torch
+import tensorflow as tf
+import tensorflow.python.ops.numpy_ops.np_config as np_config
+
+
+def torch_jac(circ):
+    def wrapper(params):
+        return torch.autograd.functional.jacobian(circ, (params))
+
+    return wrapper
+
+
+def tf_jac(circ):
+    def wrapper(params):
+        with tf.GradientTape() as tape:
+            loss = circ(params)
```

Comment:
```suggestion
    def wrapper(*args, **kwargs):
        with tf.GradientTape() as tape:
            loss = circ(*args, **kwargs)
```

this might be required for full compatibility with arbitrary QNodes!